azure-pipelines.yml

```diff
@@ -211,12 +231,12 @@ jobs:
     inputs:
       pathtoPublish: '$(Build.ArtifactStagingDirectory)'
       artifactName: 'PythonWorker'
-  - task: NuGetCommand@2
-    condition: eq(variables['UPLOADPACKAGETOPRERELEASEFEED'], true)
-    inputs:
-      command: 'push'
-      packagesToPush: '$(Build.ArtifactStagingDirectory)/**/*.nupkg;!$(Build.ArtifactStagingDirectory)/**/*.symbols.nupkg'
-      nuGetFeedType: 'internal'
-      publishVstsFeed: 'e6a70c92-4128-439f-8012-382fe78d6396/f37f760c-aebd-443e-9714-ce725cd427df'
-      allowPackageConflicts: true
-    displayName: '[Integration Test] Push NuGet package to the AzureFunctionsPreRelease feed'
+  #- task: NuGetCommand@2
+  #  condition: eq(variables['UPLOADPACKAGETOPRERELEASEFEED'], true)
+  #  inputs:
+  #    command: 'push'
+  #    packagesToPush: '$(Build.ArtifactStagingDirectory)/**/*.nupkg;!$(Build.ArtifactStagingDirectory)/**/*.symbols.nupkg'
+  #    nuGetFeedType: 'internal'
+  #    publishVstsFeed: 'e6a70c92-4128-439f-8012-382fe78d6396/f37f760c-aebd-443e-9714-ce725cd427df'
+  #    allowPackageConflicts: true
+  #  displayName: '[Integration Test] Push NuGet package to the AzureFunctionsPreRelease feed'
```

Comment:
they are commented out temporarily to test the metadata generation task, will be uncommented out once the testing is done